src/main/java/org/mockito/verification/VerificationSucceededEvent.java

```diff
@@ -0,0 +1,26 @@
+package org.mockito.verification;
+
+public class VerificationSucceededEvent {
```

Comment:
This type needs to be exposed as an interface, and not as a concrete class. It's very important that we expose all API with interfaces (with some exceptions for very good reasons, like Mockito, Answers, etc).